Lib/glyphsLib/builder/anchors.py

```diff
@@ -60,14 +62,30 @@ def _propagate_glyph_anchors(self, ufo, parent, processed):
                 base_components.append(component)
                 anchor_names |= {a.name for a in glyph.anchors}
 
+    if mark_components and not base_components and _is_ligature_mark(parent):
+        # The composite is a mark that is composed of other marks (E.g.
+        # "circumflexcomb_tildecomb"). Promote the mark that is positioned closest
+        # to the origin to a base.
+        try:
+            component = _component_closest_to_origin(mark_components, ufo)
+        except Exception as e:
+            raise Exception(
+                "Error while determining which component of composite "
+                "'{}' is the lowest: {}".format(parent.name, str(e))
+            )
+        mark_components.remove(component)
+        base_components.append(component)
+        glyph = ufo[component.baseGlyph]
+        anchor_names |= {a.name for a in glyph.anchors}
```

Comment:
Do we need to go recursive here?

Reply:
the recursive-ness is in the self-call to `_propagate_glyph_anchors` above